src/pages/githubCallback/GitHubCallback.tsx

```diff
@@ -0,0 +1,75 @@
+import * as QueryString from 'query-string';
+import { useEffect, useState } from 'react';
+
+import * as GitHubUtils from '../../features/github/GitHubUtils';
+
+/**
+ * The page that the user is redirected to after they have approved the app through GitHub.
+ * This page will complete the OAuth workflow by sending the access code the back-end to retrieve the auth-token.
+ * The auth-token is then broadcasted back to the main browser page.
+ */
+export function GitHubCallback() {
+  const [message, setMessage] = useState('You have reached the GitHub callback page');
+
+  useEffect(() => {
+    const currentAddress = window.location.search;
+    const accessCode = GitHubUtils.grabAccessCodeFromURL(currentAddress);
+
+    const clientId = GitHubUtils.getClientId();
+    const backendLink = 'https://api2.sourceacademy.nus.edu.sg/github_oauth';
```

Comment:
Please pull this out into a configuration environment variable in Constants.ts (say, GITHUB_OAUTH_PROXY_URL)

Reply:
I pulled it into Constants and updated the environment variables in 08fb3fe and 6138b1d.